.circleci/config.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - run: 'make check-deps'
       - run:
           name: "Install golangci-lint"
-          command: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.61.0
+          command: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.62.0
```

Comment:
I'm not a fan of this as it opens a security risk because now everyone with write-access to that shell can execute everything in our CircleCI env. Is this really necessary?

Reply:
Ok, reverted to `go install`